meson.build

```diff
@@ -4,7 +4,7 @@ project('classdesc', 'c',
     meson_version: '>=0.39.1',
     license: [ 'GPL3+', 'MIT' ],
     default_options: [
-        'c_std=gnu11',
+        'c_std=gnu99',
```

Comment:
This is a C++ project, and the version to target is c++98.

There is one test directory that tests classdesc compatibility with more recent C++ standards, currently set to C++11, but due to be updated to C++17 shortly.